src/main/java/org/folio/marc4ld/util/ResourceKind.java

```diff
@@ -11,7 +11,7 @@
 import org.folio.ld.dictionary.ResourceTypeDictionary;
 
 @UtilityClass
-public class IdentifierTypes {
+public class ResourceKind {
   public static final Set<ResourceTypeDictionary> BIBLIOGRAPHIC_TYPES = Set.of(INSTANCE, WORK);
```

Comment:
I think it makes sense to leave just 'BIBLIOGRAPHIC' and 'AUTHORITY':
ResourceKind.BIBLIOGRAPHIC,
ResourceKind.AUTHORITY